proto-lens-tests/proto-lens-tests.cabal

```diff
@@ -47,6 +47,26 @@ Test-Suite canonical_test
                , proto-lens-tests
                , test-framework
 
+Test-Suite discrimination_test
+  default-language: Haskell2010
+  type: exitcode-stdio-1.0
+  main-is: disc_test.hs
+  hs-source-dirs: tests
+  other-modules: Proto.Canonical
+  build-depends: base
+               , containers
+               , data-default
+               , discrimination
+               , lens-family
+               , HUnit
+               , proto-lens
+               , proto-lens-protoc
+               , proto-lens-arbitrary
+               , proto-lens-discrimination
```

Comment:
Can we move the tests to a test-suite in the proto-lens-discrimination package, similar to what we do for proto-lens-combinators?  (As in that package, it doesn't really share code (or need to) with the rest of proto-lens-tests.)

Also, along those lines: please add proto-lens-discrimination and discrimination-ieee754 to stack.yaml and travis-cabal.sh so they get tested by travis.